pkgs/tools/wayland/hyprland-iio/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, meson, cmake, pkg-config, dbus, ninja, ... }:
+
+stdenv.mkDerivation {
+  pname = "iio hyprland";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "short-circuit";
+    repo = "iio-hyprland";
+    rev = "f13242bbdf4a0d062683bf23b2431bb72e2e6ab9";
+    sha256 = "0ahy72b66f3c0a0wydbcmn8zbfr335hv9jngw0zxh097z7h9sr2y";
```

```suggestion
    hash = "0ahy72b66f3c0a0wydbcmn8zbfr335hv9jngw0zxh097z7h9sr2y";
```
Prefer `hash` over `sha256`